spikeextractors/extractors/mearecextractors/mearecextractors.py

```diff
@@ -73,12 +78,12 @@ def get_sampling_frequency(self):
     def get_traces(self, channel_ids=None, start_frame=None, end_frame=None):
         if np.any(np.diff(channel_ids) < 0):
             sorted_idx = np.argsort(channel_ids)
-            recordings = self._recordings[np.sort(channel_ids), start_frame:end_frame]
+            recordings = self._recordings[start_frame:end_frame, np.sort(channel_ids)]
```

Comment:
@samuelgarcia you need to transpose this

Reply:
It was done I was sure I push it.

Reply:
In fact, I have push it on the main repo. Oups.
i need to delete that branch.

Reply:
@samuelgarcia this still needs a transpose